lib/tasks/local_dev.rake

```diff
@@ -35,14 +35,205 @@ task setup_local_dev_data: %i[environment copy_feature_flags_from_production syn
     make_decisions: true,
   )
 
+  puts 'Creating users for providers 1N1, 24P, 1JB, D39 with various permission combinations...'
+
+  # can_view__with_safeguarding_information
```

Comment:
This is all good and useful — maybe it could be tidier in a different file, though? `CreateExampleProviderUsersWithPermissions` or sth?